src/publish.proj

```diff
@@ -24,4 +28,22 @@
     <!-- skip uploadToAzure when there are no nupkgs -->
     <CallTarget Targets="UploadToAzure" Condition="'@(_PackagesToPublish)' != ''" />
   </Target>  
+  
+  <Target Name="FinalizePublish" >
```

Comment:
+1, also there's some mixing of Push/Publish and Azure/Blob here, would be cool to be consistent for the first commit like this, otherwise both terms may get used.